tests/rustdoc-gui/collapse-trait-impl.goml

```diff
@@ -0,0 +1,23 @@
+// Checks that individual trait impls can only be collapsed via clicking directly on the summary element
+include: "utils.goml"
```

Comment:
You can remove this line, you don't use anything from this file after all. :wink: